### PR TITLE
add enrichment time series

### PR DESCRIPTION
**Pull request by rwcarlsen**

### CODE REVIEW

**Comment by gidden** [comment]
@rwcarlsen, can you add a test to `tests/toolkit/timeseries_tests.cc` please. it should be trivial.

**Comment by gidden** [comment]
Also, do we have any sort of convention for what should be added as a kernel vs. "user" time series? For the record, I am +1 for these being kernel time series and supporting a variety of time series types in the kernel.

File: src/toolkit/timeseries.cc [hunk 1]
```diff
@@ -8,6 +8,15 @@ void RecordTimeSeries<POWER>(cyclus::Agent* agent, double value) {
   RecordTimeSeries<double>("Power", agent, value);
 }
 
+template <>
+void RecordTimeSeries<ENRICH_SWU>(cyclus::Agent* agent, double value) {
```
Comment (gidden) [inline src/toolkit/timeseries.cc]: @rwcarlsen, do you have a case for naming this `ENRICH_SWU` vs. `SWU`? I know it gives semantic meaning, but in general we have eschewed semantics in favor of explicit joining in order to discover agent/prototype type.

File: src/toolkit/timeseries.cc [hunk 2]
```diff
@@ -8,6 +8,15 @@ void RecordTimeSeries<POWER>(cyclus::Agent* agent, double value) {
...
+  RecordTimeSeries<double>("EnrichmentSWU", agent, value);
+}
+
+template <>
+void RecordTimeSeries<ENRICH_FEED>(cyclus::Agent* agent, double value) {
```
Comment (gidden) [inline src/toolkit/timeseries.cc]: Whereas here `FEED` would likely be an overloaded label.

File: src/toolkit/timeseries.cc (hunk 1 above)
Comment (rwcarlsen) [inline src/toolkit/timeseries.cc]: It isn't meant to be archetype specific - it is just meant to indicate that the swu is associated with some sort of generalized material enrichment.  And this way - people know to associate the feed and swu series with each other.

Comment (gidden) [inline src/toolkit/timeseries.cc]: Ok, sounds good, thanks!

**Comment by rwcarlsen** [comment]
tests added

**Comment by gidden** [comment]
Thanks, this looks good. I'll merge when CI finishes.

**Comment by gidden** [comment]
thanks @rwcarlsen

**Pull request merged**